upgrade_test.py

```diff
@@ -161,9 +161,12 @@ def validate_truncated_entries_for_table(self, session, system_truncated=False):
                                 msg='Expected truncated entry in the system.local table, but it\'s not found')
 
     @truncate_entries
+    def upgrade_node(self, node, upgrade_sstables=True):
```

Comment:
I recall there was a comment here about the "truncate_entries" that it requires the test to have some info in that table in order for the truncate not to fail.
What was the outcome of it?

Reply:
i moved the call in this function to a common "private" function, that is what i call too, without the `truncate_entries` decorator